frontend/www/js/omegaup/components/contest/Groupsv2.vue

```diff
@@ -0,0 +1,74 @@
+<template>
+  <div class="card mt-3">
+    <div class="card-body">
+      <form class="form" @submit.prevent="$emit('emit-add-group', groupName)">
+        <div class="form-group">
+          <label>{{ T.wordsGroup }}</label>
+          <omegaup-autocomplete
+            v-model="groupName"
+            :init="(el) => typeahead.groupTypeahead(el)"
+          ></omegaup-autocomplete>
+        </div>
+        <button class="btn btn-primary" type="submit">
+          {{ T.contestAddgroupAddGroup }}
+        </button>
+      </form>
+    </div>
+    <table class="table table-striped mb-0">
+      <thead>
+        <tr>
+          <th class="text-center">
+            {{ T.contestEditRegisteredGroupAdminName }}
+          </th>
+          <th class="text-center">{{ T.contestEditRegisteredAdminDelete }}</th>
+        </tr>
+      </thead>
+      <tbody>
+        <tr v-for="group in groups" :key="group.alias">
+          <td>
+            <a :href="`/group/${group.alias}/edit/`">{{ group.name }}</a>
+          </td>
+          <td class="text-center">
+            <button
+              class="close float-none"
+              type="button"
+              @click="$emit('emit-remove-group', group.alias)"
+            >
+              ×
+            </button>
+          </td>
+        </tr>
+      </tbody>
+    </table>
+  </div>
+</template>
+
+<script lang="ts">
+import { Vue, Component, Prop, Watch } from 'vue-property-decorator';
+import { types } from '../../api_types';
+import T from '../../lang';
+import * as typeahead from '../../typeahead';
+import Autocomplete from '../Autocomplete.vue';
+
+@Component({
+  components: {
+    'omegaup-autocomplete': Autocomplete,
+  },
+})
+export default class Groups extends Vue {
+  @Prop() initialGroups!: types.ContestGroup[];
```

Comment:
ahora estamos intentando que los props ya no tengan ningún prefijo. si el componente padre necesita acceder esto, preferimos usar `.sync` (aunque [está deprecado en v3](https://v3.vuejs.org/guide/migration/v-model.html#_2-x-syntax)).

```suggestion
  @Prop() groups!: types.ContestGroup[];
```

L64 se puede eliminar porque si el componente padre modifica `groups`, nos va a notificar y se va a sincronizar. puedes seguir cómo se está implementando en [`Groups.vue`](https://github.com/omegaup/omegaup/blob/master/frontend/www/js/omegaup/components/contest/Groups.vue).

Reply:
No me quedó del todo claro si debo de dejar de usar `Watch` o no. `.sync` no ayuda en ese sentido. Y sin el watch, deja de ser reactivo.

Reply:
aaaah el sync es sólo si el componente padre _necesita_ ser notificado de cambios. en este caso no es necesario!